File: _src/02/Chapter2.CSharp/Initializer.cs
using System.Data.Entity;

namespace Chapter2.CSharp
{
    public class Initializer : DropCreateDatabaseIfModelChanges<Context>
    {
        protected override void Seed(Context context)
        {
            context.Companies.Add(new Company
            {
                Name = "My company"
            });
        }
    }
}

File: _src/02/Chapter2.VB/Initializer.vb
Imports System.Data.Entity

Public Class Initializer
    Inherits DropCreateDatabaseIfModelChanges(Of Context)

    Protected Overrides Sub Seed(ByVal context As Context)
        context.Companies.Add(New Company() With {
             .Name = "My company"
        })
    End Sub
End Class
